packages/math-input/src/components/tabbar/item.tsx

```diff
@@ -7,6 +7,7 @@ import * as React from "react";
 import IconAsset from "./icons";
 
 import type {KeypadPageType} from "../../types";
```

Comment:
All of these GitHub Actions errors makes me think your IDE might be messed up; I think a lot of these could be auto-fixed for you by your IDE. LMK if you want help troubleshooting.